routes/user.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const router = express.Router();
+const User = require("../models/User");
+
+// on first login reset Password
+router.post("/:hashMail", async (req, res) => {
+  let { password } = req.body;
+  let { hashMail } = req.params;
+  if (hashMail) {
+    try {
+      const user = await User.findOne({ hashMail });
+      user.password = password;
+      delete user.hashMail;
+      const UpdatedUser = await user.save();
```

Comment:
Should be updatedUser and not UpdatedUser. Caps is used only for global variables.